ui/components/AlertsTable.tsx

```diff
@@ -103,12 +116,23 @@ function AlertsTable({ className, rows = [] }: Props) {
   ];
 
   return (
-    <DataTable
-      className={className}
-      fields={alertFields}
-      rows={rows}
-      filters={initialFilterState}
-    />
+    <>
+      <DataTable
+        className={className}
+        fields={alertFields}
+        rows={rows}
+        filters={initialFilterState}
+      />
+      <Dialog open={yamlView !== null} onClose={() => setYamlView(null)}>
+        {yamlView && (
+          <YamlView
+            object={yamlView}
+            yaml={yamlView?.yaml}
+            className="dialog"
```

Comment:
Same here: passing internal className from the outside. It would be better to use a boolean prop (`isDialog`?) for this.